geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -184,11 +184,13 @@ object RichSimpleFeatureType {
 
   import scala.collection.JavaConversions._
 
+  val GEOMESA_PREFIX      = "geomesa."
   val SCHEMA_VERSION_KEY  = "geomesa.version"
   val TABLE_SHARING_KEY   = "geomesa.table.sharing"
   val SHARING_PREFIX_KEY  = "geomesa.table.sharing.prefix"
   val DEFAULT_DATE_KEY    = "geomesa.index.dtg"
   val ST_INDEX_SCHEMA_KEY = "geomesa.index.st.schema"
+  val USER_DATA_PREFIX    = "geomesa.user-data.prefix"
```

Comment:
There's a unit test here: https://github.com/locationtech/geomesa/pull/888/files/abab8d9385eb9178b7f410458e288cc2ce6ab05f#diff-c080263f8b3ca242a804fb80412668d6R57
Do you mean in the docs? no...